packages/gatsby-cli/src/reporter/index.ts

```diff
@@ -0,0 +1,11 @@
+import { startLogger } from "./start-logger"
+import { patchConsole } from "./patch-console"
+import { catchExitSignals } from "./catch-exit-signals"
+import { reporter } from "./reporter"
+
+catchExitSignals()
+startLogger()
+patchConsole(reporter)
+
+export default reporter
```

Comment:
Can you leave a comment why this double export is needed at the moment?

Reply:
removed it!

Reply:
@blainekasten have you pushed the changes?